src/Conf.ml

```diff
@@ -1777,6 +1784,8 @@ let read_config_file conf filename_kind =
                 let from = `File (filename, num) in
                 match parse_line conf ~from line with
                 | Ok conf -> (conf, errors, Int.succ num)
+                | Error _ when !ignore_invalid_options ->
+                    (conf, errors, Int.succ num)
```

Comment:
Should we warn in this case ? Some users (eg. editor plugins) may want to always enable this but still show warnings to the user.

Reply:
Good point.

The `don't warn if quiet` bit doesn't work well as the command line (containing `--quiet`) is parsed after the files. Any suggestion on how to fix this ?

Reply:
Command line arguments that are not formatting options (eg. `inplace`, `debug`) are parsed before but `quiet` is an exception because it can be set in `.ocamlformat` files.
Maybe we could set `conf.quiet` both before and after parsing files and env (`quiet` can be set in env too) ? Otherwise, we have to accumulate warnings and print them after.